cncf/settings.yml

```diff
@@ -217,6 +217,9 @@ groups:
   - name: Wasm
     categories:
       - Wasm
+  - name: Cloud Native Artificial Intelligence
```

Comment:
Could you use CNAI as the group name as well (or any other shorter version you prefer)? The reason is that this name is displayed in the group selector, and the one you provided is quite long 😅

<img width="970" alt="Screenshot 2024-05-10 at 17 33 01" src="https://github.com/cncf/landscape2-sites/assets/1213902/47d7c1d6-ed28-46a7-9440-c94376d7adad">